pre_commit_hooks/check_merge_conflict.py

```diff
@@ -10,15 +10,25 @@
     b'=======\n',
     b'>>>>>>> ',
 ]
+GITDIR_PREFIX_LEN = len('gitdir: ')
```

Comment:
I would prefer to use git's mechanisms for discovering this directory -- using implementation detail like reading the file is going to be fragile